glide.diff

```diff
@@ -0,0 +1,72 @@
+diff --no-dereference -N -r current/vendor/github.com/openshift/api/servicecertsigner/v1alpha1/types.go updated/vendor/github.com/openshift/api/servicecertsigner/v1alpha1/types.go
```

Comment:
This was meant to be pushed - the commits associated with it are marked `DO NOT MERGE` because as per [my previous comment](https://github.com/openshift/service-ca-operator/pull/73#issuecomment-543607442) changes need to merge to deps before this PR will be mergeable.